scripts/version.sh

```
@@ -0,0 +1,153 @@
+  local env=${3:-"prod"}
+  
+  case "$command" in
+    get)
+      get_current_version
+      ;;
+    bump)
+      if [ -z "$version_type" ]; then
+        echo "Error: Please specify a version type (major, minor, patch)" >&2
+        exit 1
+      fi
+      local new_version=$(generate_version "$version_type")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)